components/rmrk/Create/CreateCollection.vue

```diff
@@ -192,6 +194,7 @@ export default class CreateCollection extends mixins(
             `[Collection] Saved ${this.base.name} in block ${blockNumber}`,
             notificationTypes.success
           )
+          this.$emit('created')
```

Comment:
Doesn't this need to be in callback success?

Reply:
I don't think so because `howAboutToExecute` is a utils method which is used in many ohter cases, like buying nft, listing nft and minting nft etc. 
Hense this event should be emitted in the `createCollection` component.

Reply:
since it's an async function, isn't there a risk of `this.$emit('created')` being called first?

Reply:
I mean this line is already in the success callback. 👀 

It will be called after the transaction is done successfully.

Reply:
Or am I missing something?

Reply:
now that I've open the full file I can see that `this.$emit('created')` is in callback success
sorry for the unnecessary intervention 🥲